frameworks/helloworld/universe/marathon.json.mustache

```diff
@@ -63,7 +63,8 @@
     "WORLD_SECRET3" : "{{world.secret3}}",
     {{/world.secret3}}
 
-    "MESOS_API_VERSION": "{{service.mesos_api_version}}"
+    "MESOS_API_VERSION": "{{service.mesos_api_version}}",
+    "ENABLED": "{{service.enabled}}"
```

Comment:
maybe a more specific env name given that this value is going to be present for all hello-world